nil/core-concepts/shards-parallel-execution.mdx

```diff
@@ -0,0 +1,49 @@
+# Shards and parallelized execution
+
+## Definition
+
+Shards are smaller chains that are usually responsible for managing a portion of the global state of the main chain. In =nil;, there exist two types of shards.
+
+* Execution shards process transactions of the accounts allocated to these shards
+* The consensus shard coordinates execution shards and submits ZKPs of the global state to Ethereum
```

Comment:
The consensus shard doesn't submit proofs on its own. It's up to sync committee node

Reply:
Yes, you are right. I have rephrased this